packages/plugin-devtools/src/components/InfoComponent.ts

```diff
@@ -34,6 +35,31 @@ export class InfoComponent extends OwlUIComponent<{}> {
     className(vNode: VNode): string {
         return vNode.constructor.name;
     }
+    /**
+     * Return an object representing the given VNode's public properties as
+     * alphabetically sorted pairs of key and value strings.
+     *
+     * @param vNode
+     */
+    nodeCustomProperties(vNode: VNode): { [key: string]: string }[] {
+        return Object.keys(vNode)
+            .filter(key => !key.startsWith('_') && !vNodeKeys.includes(key))
+            .sort()
+            .map(key => {
+                let value = '' + vNode[key];
+                if (typeof vNode[key] === 'object') {
+                    if (!vNode[key] || !Object.keys(vNode[key]).length) {
```

Comment:
This implies that null, set, array becomes '{}', is it wanted?

Reply:
Nope. This is indeed incomplete.
Is this better?
```typescript
if (Array.isArray(vNode[key]) && !vNode[key].length) {
    value = '[]';
} else if (vNode[key] === null) {
    value = 'null';
} else if (vNode[key] === undefined) {
    value = 'undefined';
} else if (vNode[key] instanceof Set) {
    const items = [];
    VNode[key].forEach(item => {
        items.push(item);
    });
    value = items.join('\n');
} else if (!vNode[key] || !Object.keys(vNode[key]).length) {
    value = '{}';
} else if (vNode[key].toString === {}.toString) {
    value = this._objectRepr(vNode[key]);
}
```